app/controllers/users/email_confirmations_controller.rb

```diff
@@ -90,6 +90,18 @@ def message_for_already_confirmed_user
       end
     end
 
+    def flash_message_for_successful_signed_in_confirmation
+      if IdentityConfig.store.feature_select_email_to_share_enabled &&
+         params[:from_select_email_flow].to_s == 'true'
```

Comment:
I haven't tested yet, but if we're relying on a parameter, does this alert banner work correctly if the user has to sign in first? From the designs, I think it's expected this would still display.

Reply:
The user wouldn't be directed through this controller after they sign in. This user would only be directed to account page in general. I can make it so that we store the attribute in session. and delete. but I think adding this to the session, and ensuring that when user lands on account page it also shows the message makes sense.